requirements.txt

```diff
@@ -0,0 +1,20 @@
+configparser==3.5.0
+docopt==0.6.2
+file-magic==0.3.0
+jsonschema==2.5.1
+oauthlib==1.0.3
+pkg-resources==0.0.0
```

Comment:
That package is a dependency of one of the packages we are installing from a tarball and is installed automatically.  I've removed it from requirements.txt.